src/app/components/ImageCard/ImageCard.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+
+interface ImageCardProps {
+  imageUrl: string;
+  imageAlt: string;
+  children: React.ReactNode;
+  imageFirst?: boolean;
```

Comment:
I gave `imageFirst={false}` a try out of curiosity. Is this the expected outcome?
<img width="1330" alt="image" src="https://github.com/user-attachments/assets/6afc1e2f-195f-4da9-a60e-b0a5d645df09">

Reply:
<img width="1702" alt="image" src="https://github.com/user-attachments/assets/5393d7e4-b85c-4a6d-88a7-c22d18eff1d9">